lib/boxen/config.rb

```diff
@@ -25,8 +25,12 @@ def self.load(&block)
           end
         end
 
-        keychain        = Boxen::Keychain.new config.user
-        config.token    = keychain.token
+        if Boxen::Util.osx?
```

Comment:
You needed `require "boxen/util"` at the top. I am getting `uninitialized constant Boxen::Util` otherwise.

```
/opt/boxen/repo/.bundle/ruby/2.3.0/gems/boxen-3.1.1/lib/boxen/config.rb:28:in `block in load': uninitialized constant Boxen::Util (NameError)
	from /opt/boxen/repo/.bundle/ruby/2.3.0/gems/boxen-3.1.1/lib/boxen/config.rb:90:in `initialize'
	from /opt/boxen/repo/.bundle/ruby/2.3.0/gems/boxen-3.1.1/lib/boxen/config.rb:15:in `new'
	from /opt/boxen/repo/.bundle/ruby/2.3.0/gems/boxen-3.1.1/lib/boxen/config.rb:15:in `load'
	from /opt/boxen/bin/boxen-git-credential:31:in `<main>'
```

Reply:
@komapa Thanks for the report! Would you mind filing a PR for this?